common/src/main/java/org/conscrypt/NativeCrypto.java

```diff
@@ -1113,6 +1113,19 @@ void verifyCertificateChain(long[] certificateChainRefs, String authMethod)
                 throws CertificateException;
 
         /**
+         * Verify that we trust the certificate chain is trusted. This is an alternative
+         * to {@link #verifyCertificateChain(long[], String)}, which is given the fully encoded
```

Comment:
This comment must be ambiguous because it doesn't make sense the way I'm reading it. `long[]` is an array of references to X509 BoringSSL instances and not to fully-encoded certificates. Did you mean that this method is different from `long[]` because it receives fully-encoded certificates and not references to native objects?